assets/RNAseq_report.Rmd

```diff
@@ -2140,10 +2187,12 @@ as.character(params$revision),
 "` pipeline [^5], which was written using the nf-core template [@ewels2020nf]. For differential expression analysis, the read quantification data resulting from `",
```

Comment:
Is this `[^5]` the citation number?

Reply:
If so, it would be missing for the other citations like `@love2014...`

Reply:
yep, it adds a superscript number and also links it to the citation

Reply:
ah sorry, no, some citations are just added without the reference thing. The reference is only done for links, i.e. [^5] moves the report down to this link: https://github.com/qbic-pipelines/rnadeseq. The other citations like love2014 are just listed without being clickable

Reply:
Is there a reason why this is not consistent with the clickable references?